openpype/pipeline/colorspace.py

```diff
@@ -221,8 +323,46 @@ def get_data_subprocess(config_path, data_type):
             "config", data_type,
             "--in_path", config_path,
             "--out_path", tmp_json_path
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
```

Comment:
This is a debug log.
```suggestion
        log.debug("Executing: {}".format(" ".join(args)))
```